mobile-widgets/qml/DiveDetailsEdit.qml

```diff
@@ -378,7 +379,7 @@ Item {
 			}
 			Controls.TextField {
 				id: txtGasMix0
-				text: usedGas[0]
+				text: usedGas[0] != null ? usedGas[0] : null
```

Comment:
How is this different from _text: usedGas[0]_?

If usedGas[0] is null you assigning null, otherwise usedGas[0].